packages/sonataflow-image-common/resources/modules/kogito-maven/common/configure

```diff
@@ -37,7 +37,7 @@ else
     # Installs Maven Wrapper for version 3.8.6 in order to be compatible with the quarkus-maven-plugin minimum requirements (on Quarkus 3.15.3).
     echo "---> Installing Maven wrapper for Maven 3.8.6"
     cd "${MAVEN_HOME}"/bin
-    mvn wrapper:wrapper -Dmaven=3.8.6 -Dtype=source
+    cp -rpTv "${SCRIPT_DIR}"/maven-wrapper/ .
```

Comment:
For the records I created `maven-wrapper` folder in ${SCRIPT_DIR} and not inside `ADDED` folder, otherwise I would have also to modify the `cp` on line 47 of this file. 

So I wanted to modify the less possible number of code lines from upstream.